crypto/default/src/main/java/org/keycloak/crypto/def/BCUserIdentityExtractorProvider.java

```diff
@@ -162,7 +164,7 @@ public Object extractUserIdentity(X509Certificate[] certs) {
                                     tempOid = oid.getId();
 
                                     ASN1Encodable principalNameEncoded = asn1Sequence.getObjectAt(1);
-                                    DERUTF8String principalName = DERUTF8String.getInstance(unwrap(principalNameEncoded));
```

Comment:
Minor: looks that `DERUTF8String` can be removed from the imports of this class?